automatic/videoder/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,23 @@
+﻿$ErrorActionPreference = 'Stop'
+
+$toolsPath   = Split-Path $MyInvocation.MyCommand.Definition
+
+$packageArgs = @{
+  packageName    = 'videoder'
+  fileType       = 'exe'
+  file           = Get-Item "$toolsPath\*Videoder Setup*.exe"
+  file64         = Get-Item "$toolsPath\*Videoder Setup*.exe"
+  silentArgs     = '/S'
+  validExitCodes = @(0)
+  softwareName   = 'videoder*'
+}
+Install-ChocolateyInstallPackage @packageArgs
+Get-ChildItem $toolsPath\*.exe | ForEach-Object { Remove-Item $_ -ea 0; if (Test-Path $_) { Set-Content "$_.ignore" '' }}
+
+$packageName = $packageArgs.packageName
+$installLocation = Get-AppInstallLocation $packageName
+if (!$installLocation)  { Write-Warning "Can't find $PackageName install location"; return }
+Write-Host "$packageName installed to '$installLocation'"
+
+Write-Host 'Adding to PATH if needed'
```

Comment:
Do we really need this tool on PATH ? Please remove this, path is limited. You can add shim if you like instead.